### ISSUE TITLE
Recurring content to plan

### ISSUE THREAD

**Issue by Cotabe** [Giveth/giveth-planning#848]
Hey as agreed on the communications call. 

Here are a few things that we can plan to have as recurring content:

- Verified makers (I saw this week we had it :)
- Platform updates (maybe every month)
- Key monthly stats (maybe stats and stories of impact)
- Regen Farms PSA
- Farms APR (already done)

**Comment by Cotabe** [comment]
@c0ric0ri  @ahmadabugosh

**Comment by ahmadabugosh** [comment]
I like it! How can we action it on Twitter @c0ric0ri ? Maybe different people can help compiling each one on a recurring basis?

**Comment by c0ric0ri** [comment]
@ahmadabugosh yes I like the idea of distributing it across different people! Then we have them flow through github for review and added to the content planner.

**Comment by ahmadabugosh** [comment]
@SnakeGH suggested that we also add recurring content for "top mover" projects. For example, projects that are trending, have a lot of donations that GIVbacks round etc. I think this would be even more relevant with GIVpower launching + we can revive the "[project highlights](https://medium.com/giveth/community-project-spotlight-2-dc0c5ca4757d)" we used to do.

**Comment by SnakeGH** [comment]
Thanks @ahmadabugosh, not only top movers but also "projects that need our help" so we don't just promote the top, but also root for the underdog :)

**Comment by c0ric0ri** [comment]
Input here from @oyealmond, content about:

- humans behind Giveth, things about the DAO, why we do things, Praise

**Comment by laurenluz** [comment]
we should also have some regular repeats of the same, but core/important ideas like:
- public goods explainers & how web3 can help
- GIVbacks
- verification
- GIVpower

**Comment by oyealmond** [comment]
Some recurring that are already happening: 
- AMA invitation to the community (Wednesdays) 
- Twitter Spaces announcements (every two weeks)

**Comment by clara-zi** [comment]
> @SnakeGH suggested that we also add recurring content for "top mover" projects. For example, projects that are trending, have a lot of donations that GIVbacks round etc. I think this would be even more relevant with GIVpower launching + we can revive the "[project highlights](https://medium.com/giveth/community-project-spotlight-2-dc0c5ca4757d)" we used to do.

Been doing project highlights for the short while I've been covering FB / LI - I could take something like this @c0ric0ri - is this something to add to epic?
